### ISSUE TITLE
About the configuration of two fine-tuning strategies in your code.

### ISSUE THREAD

**Issue by BladeCoda** [linwhitehat/ET-BERT#4]
While reading you paper, we noted that the packet-level fine-tuning outperform the flow-level one in most tasks.

Could you tell us how to use the packet-level fine-tuning in this code for comparison,  since we saw that you compared these two fine-tuning strategies is every task. However, we didn't find a specific configuration of selecting these two strategies in the file of "run_classification.py".

Or, these two strategies are about different data processing for raw traffic?

Thank you.

**Comment by linwhitehat** [comment]
> While reading you paper, we noted that the packet-level fine-tuning outperform the flow-level one in most tasks.
> 
> Could you tell us how to use the packet-level fine-tuning in this code for comparison, since we saw that you compared these two fine-tuning strategies is every task. However, we didn't find a specific configuration of selecting these two strategies in the file of "run_classification.py".
> 
> Or, these two strategies are about different data processing for raw traffic?
> 
> Thank you.

Thank you for your interest in our work.

In the fine-tuning phase, the differences between the packet-level and flow-level strategies are specifically the differences in input data and parameters.
I have updated the code to just use the data processed at the packet level and set `seq_length` when fine-tuning, then we set the parameters at the flow level and the packet level to 512 and 128.

`python3 fine-tuning/run_classifier.py --pretrained_model_path models/pre-trained_model.bin \
                                   --vocab_path models/encryptd_vocab.txt \
                                   --train_path datasets/cstnet-tls1.3/packet/train_dataset.tsv \
                                   --dev_path datasets/cstnet-tls1.3/packet/valid_dataset.tsv \
                                   --test_path datasets/cstnet-tls1.3/packet/test_dataset.tsv \
                                   --epochs_num 10 --batch_size 32 --embedding word_pos_seg \
                                   --encoder transformer --mask fully_visible \
                                   --seq_length 128 --learning_rate 2e-5`
